src/librustc_ast/lib.rs

```diff
@@ -1,4 +1,22 @@
-//! The Rust parser and macro expander.
+//! The Rust abstract syntax tree module.
```

Comment:
Instead of doing this, please reexport everything in `rustc_ast::ast` as in the `lib.rs` file.
This way we avoid losing all the git history and we separate the data-types from everything else.

Reply:
Eugh, we were hoping it would show up as a rename from `ast.rs` to `lib.rs`, this `git` failure mode is annoying.

Also, there's no "everything else", AFAIK there's just data types and sub modules.

Reply:
There are a bunch of submodules there yes, but those are less drowned out if they have their own file in `lib.rs`. This is how `rustc_hir` is organized, and it works great.

Reply:
I agree with Centril that it would be good if everything that previously was in `ast.rs` stayed in `ast.rs`.
(I'm also ok with the status quo and `rustc_ast::ast` doesn't bother me.)

Reply:
Should be fixed now.